### PR TITLE
Fixed image drag and drop

### PR DESCRIPTION
**Pull request by sankethgunasekara**
Changed the compiler options to target es5 to es6 to overcome the type error.

### CODE REVIEW

File: PCTPCF/src/RichTextEditor.tsx [hunk 1]
```diff
@@ -138,12 +138,12 @@ function Internal({ value, onBlur, placeholder, minHeight = 100, maxHeight, disa
             <AlignmentButton format="center" disabled={disabled} />
             <AlignmentButton format="right" disabled={disabled} />
           </Flex>
-          {/* <WhenInRole SuperAdmin> */}
+          {/* <WhenInRole SuperAdmin>
```
Comment (tilangapramithvidusara) [inline PCTPCF/src/RichTextEditor.tsx:141]: If don't need this code block then remove them

Comment (sankethgunasekara) [reply PCTPCF/src/RichTextEditor.tsx:141]: It was on the original code therefore kept it as comment

Comment (tilangapramithvidusara) [reply PCTPCF/src/RichTextEditor.tsx:141]: it is for check user role. in your code do you have manage permissions according to user role

Comment (sankethgunasekara) [reply PCTPCF/src/RichTextEditor.tsx:141]: As of Scotts code they had that. But for the documentation of "Rich Text Editor PCF for Dataverse" they have mentioned that " Roles affected
Seer Admin
Partner Admin
Partner Consultant"
So should I remove the code segment ?

Comment (tilangapramithvidusara) [reply PCTPCF/src/RichTextEditor.tsx:141]: okay then keep it up

**Review by tilangapramithvidusara — APPROVED** [review]

**Pull request merged**